### DISCUSSION TITLE
when

### DISCUSSION THREAD

**Discussion by nito9999** [RealMCoded/LunarRec#8] [Q&A] [answered]
whens the guide for setup

**Comment by RealMCoded** [comment, marked as answer]
For now, follow the guide on the Rec.JS setup guide since the steps are mostly the same. Do note however the C/C++ build tools should also be installed due to the servers reliance on sqlite3. 

https://github.com/nodejs/node-gyp make sure to follow the setup instructions as well since you need to install Python and Microsoft build tools
